package.json

```diff
@@ -23,7 +23,9 @@
     "clean": "rimraf target && rimraf dist && rimraf build2",
     "lint:report": "grunt lint --checkstyle",
     "lint": "grunt lint",
-    "test": "grunt test",
+    "test": "grunt generate-config && karma start",
```

Comment:
`yarn test` is intend for local development hence keeping it running.

Reply:
Gotcha. Just so I understand what you're thinking: we're going to push devs to run `yarn test` when they're developing locally and/or debugging tests. Keeping the karma-runner open should allow them to update tests in real time, where the `autoWatch` option will execute tests on file changes.

The idea of keeping the test-runner open after running `yarn test` seems strange to me, as I would expect a `test` script to exit with a `0` or `1` exit code. However, if this is common in other libraries I'm happy to add it.

As an alternative, I have seen this option as a "debug" mode in other libraries - maybe something like `test:dev` would be more clear for other developers?

Reply:
I don't have strong option either way actually

This is what we have in okta-ui today (cd to any admin package and run `yarn test`)
Since admin package is usually relatively small hence keep the watch mode may make more sense.
Kind of trying to promote the authWatch mode and hopefully the experience we'd have would be
1. `yarn test` : change any source and test will auto rerun but it takes some time (2mins)
2. `yarn test --test SomeFile_spec.js` run single test spec. shall be way faster than option1
3. `yarn test` then use the `Expect.fitp` to run single test case, which shall be super fast.

The alternative we could have is basically swap the `test` and `test:report`
- `yarn test` run test once and exit
- `yarn test:dev` run test and keep watching

we could easily change to one way or another as long as we're more comfortable.

Reply:
Personally I don't like that the test runner does not exit after yarn test. I don't mind having 
yarn test run test once and exit
yarn test:dev run test and keep watching

Reply:
add another commit to make the change.